code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -286,7 +286,7 @@
 
 /obj/item/storage/fancy/cigar
 	name = "cigar case"
-	desc = "A case for holding your cigars when you are not smoking them. Fancy!"
+	desc = "A case for holding your cigars when you are not smoking them. Fancy! Alt+click to open and close."
```

Comment:
Bruh, there are no things like these in the description, do not put it here.

Reply:
How to tell about alt+clicking then? Hardcases have it in their desc.